contracts/import.sol

```diff
@@ -0,0 +1,5 @@
+// SPDX-License-Identifier:MIT
+pragma solidity ^0.6.12;
+
+import '@rsksmart/rif-relay-contracts/dist/contracts/smartwallet/SmartWallet.sol';
```

Comment:
shouldn't this contracts be included outside the dist folder?

Reply:
Yes, I think it would be much better indeed, thank you for pointing it out